### ISSUE TITLE
SMAPI Android installer 3.2.0.2 wont start

### ISSUE THREAD

**Issue by Gaaraszauber** [MartyrPher/SMAPI-Android-Installer#56]
If i start the installed app after few seconds the app closed its self.
device: Samsung galaxy s5 neo
android-version: 6.0.1

**Comment by MartyrPher** [comment]
The issue was slowly determined to be somewhere within the ConfigEditorFragment. Waiting on an update on the latest build.

**Comment by MartyrPher** [comment]
This was solved. The images for the background were too large of a resolution for certain phones. Down-scaling those background images fixed the issue.

**Comment by heyloony** [comment]
Same problem on a LGG4 android 6. It crashes after a few seconds.

**Comment by MartyrPher** [comment]
This issue is fixed in 3.2.0.3. If the issue persists, please feel free to open a new issue.
